pkg/storage/jsindexeddb/jsindexeddb.go

```diff
@@ -217,3 +256,17 @@ func getResult(req js.Value) (*js.Value, error) {
 		return nil, errors.New("timeout waiting for eve")
 	}
 }
+
+// since jsindexdb doesn't support adding object stores on fly, using predefined object store names to
+//  create object store in advance instead of creating a database per store.
+func getStoreNames() []string {
```

Comment:
this array should be collected at a higher layer package and passed-in as an option.

Reply:
it may require interface change, added a TODO and created issue #1347